MediaPicker/src/main/java/org/wordpress/android/util/ImageUtils.kt

```diff
@@ -0,0 +1,84 @@
+package org.wordpress.android.util
+
+import android.widget.ImageView
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.appcompat.content.res.AppCompatResources
+import androidx.core.content.ContextCompat
+import androidx.core.widget.ImageViewCompat
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import kotlinx.coroutines.CoroutineScope
+import org.wordpress.android.mediapicker.R
+
+fun ImageView.setImageResourceWithTint(
+    @DrawableRes drawableResId: Int,
+    @ColorRes colorResId: Int
+) {
+    setImageDrawable(ContextCompat.getDrawable(context, drawableResId))
+    ImageViewCompat.setImageTintList(
+        this,
+        AppCompatResources.getColorStateList(context, colorResId)
+    )
+}
+
+fun ImageView.announceSelectedImageForAccessibility(itemSelected: Boolean) {
+    if (itemSelected) {
+        announceForAccessibility(
+            context.getString(R.string.photo_picker_image_thumbnail_selected)
+        )
+    } else {
+        announceForAccessibility(
+            context.getString(R.string.photo_picker_image_thumbnail_unselected)
+        )
+    }
+}
+
+fun ImageView.cancelRequestAndClearImageView() {
+    Glide.with(context).clear(this)
+}
+
+fun ImageView.load(
+    imgUrl: String,
+    @DrawableRes drawableResId: Int
+) {
+    Glide.with(context)
+        .load(imgUrl)
+        .error(drawableResId)
+        .placeholder(drawableResId)
+        .fitCenter()    // TODO: we might need to make this customisable based on the needs
+        .into(this)
+        .clearOnDetach()
+}
+
+fun ImageView.loadThumbnailFromVideoUrl(
+    coroutineScope: CoroutineScope,
+    videoUrl: String
+) {
+    // TODO: Anitaa: we would need to find a way to decouple VideoLoader since it uses FluxC internally
```

Comment:
True, we might want to extract a new module in `FluxC` or have a bridge similar to `MimeTypeSupportProvider`.